Netable/Example/MockNetworkingService/JSONData/login.json

```diff
@@ -0,0 +1,5 @@
+{
+    "email": "cat@netable.com",
```

Comment:
Very minor point, but we probably shouldn’t use a potentially real domain that we don’t control and isn’t a known sample domain for something like this. I’d be inclined to just use `example.com` here and anywhere else we are using this right now.